api/src/Entity/MaterialItem.php

```diff
@@ -110,6 +110,7 @@ class MaterialItem extends BaseEntity implements BelongsToCampInterface, CopyFro
      * The number of items or the amount in the unit of items that are required.
      */
     #[ApiProperty(example: 1.5)]
+    #[Assert\GreaterThan(0)]
```

Comment:
are there any implications for existing (now invalid) data?

Reply:
yes, existing values with invalid content cannot be saved anymore until you make a valid value out of it